tests/unit/BcolErrorDialog.spec.ts

```diff
@@ -15,7 +13,7 @@ describe('BcolErrorDialog - Verify parameters passed in are displayed correctly'
     const wrapper = shallowMount(BcolErrorDialog,
       {
         propsData: {
-          filingType: FilingTypes.ANNUAL_REPORT,
+          filingType: 'annualReport',
```

Comment:
Tests should use hard-coded types, so that they break if you change the type and forget to update the tests.